mappings/net/minecraft/block/PowderSnowBlock.mapping

```diff
@@ -1,4 +1,10 @@
 CLASS net/minecraft/unmapped/C_rrackgmo net/minecraft/block/PowderSnowBlock
+	FIELD f_dgnnnwym HORIZONTAL_PARTICLE_MOMENTUM_FACTOR F
+	FIELD f_mzutkeqq IN_BLOCK_HORIZONTAL_SPEED_MULTIPLIER F
+	FIELD f_ormapkca MINIMUM_FALL_DISTANCE_FOR_SOUND D
+	FIELD f_sxazzfdg IN_BLOCK_VERTICAL_SPEED_MULTIPLIER F
+	FIELD f_vaourciu NUM_BLOCKS_TO_FALL_INTO_BLOCK F
 	FIELD f_yjwkdtzv FALLING_SHAPE Lnet/minecraft/unmapped/C_zscvhwbd;
+	FIELD f_yylnwwdr MINIMUM_FALL_DISTANCE_FOR_BIG_SOUND D
```

Comment:
None of these seem like awfully good names, not that I have any better suggestions though